### PR TITLE
Add 2022.2.1 and 2022.2.2 to NVDA API versions

### PR DESCRIPTION
**Pull request by seanbudd**
Afterwards: Re-run the last "Transform NVDA addons to views" on [addon-datastore](https://github.com/nvaccess/addon-datastore/actions) to regenerate projections (views) for the add-on datastore API.

### CODE REVIEW

File: nvdaAPIVersions.json [hunk 1]
```diff
@@ -206,5 +206,18 @@
...
+		"apiVer": {
+			"major": 2022,
+			"minor": 2,
+			"patch": 1
+		},
+		"backCompatTo": {
+			"major": 2022,
+			"minor": 1,
+			"patch": 0
+		}
```
Comment (seanbudd) [inline nvdaAPIVersions.json:221]:
```suggestion
		},
	{
		"description": "NVDA 2022.2.2",
		"apiVer": {
			"major": 2022,
			"minor": 2,
			"patch": 2
		},
		"backCompatTo": {
			"major": 2022,
			"minor": 1,
			"patch": 0
		}
	}
```

File: nvdaAPIVersions.json [hunk 2]
```diff
@@ -206,5 +206,31 @@
...
+			"major": 2022,
+			"minor": 2,
+			"patch": 2
+		},
+		"backCompatTo": {
+			"major": 2022,
+			"minor": 1,
+			"patch": 0
+		}
+	}
```
Comment (seanbudd) [inline nvdaAPIVersions.json:234]:
```suggestion
```

**Review by feerrenrut — APPROVED** [review]
Looks good

**Comment by feerrenrut** [comment]
We'll need to look into why the tests a re failing here.

File: nvdaAPIVersions.json [hunk 3]
```diff
@@ -206,5 +206,30 @@
...
+			"major": 2022,
+			"minor": 2,
+			"patch": 1
+		},
+		"backCompatTo": {
+			"major": 2022,
+			"minor": 1,
+			"patch": 0
+		},
+	{
```
Comment (seanbudd) [inline nvdaAPIVersions.json:222]: fix up missing brace

```suggestion
		},
	},
	{
```

File: nvdaAPIVersions.json [hunk 4]
```diff
@@ -206,5 +206,31 @@
...
+		"apiVer": {
+			"major": 2022,
+			"minor": 2,
+			"patch": 1
+		},
+		"backCompatTo": {
+			"major": 2022,
+			"minor": 1,
+			"patch": 0
+		},
```
Comment (seanbudd) [inline nvdaAPIVersions.json:221]: remove extra comma

```suggestion
		}
```

**Comment by seanbudd** [comment]
fixed the failing tests

**Pull request merged**